internal/client/influxdb/influxdb.go

```diff
@@ -0,0 +1,51 @@
+package influxdb
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/influxdata/influxdb/client/v2"
+	"github.com/mainflux/mainflux/internal/env"
+	"github.com/mainflux/mainflux/pkg/errors"
+)
+
+var (
+	errConnect = errors.New("failed to create InfluxDB client")
+	errConfig  = errors.New("failed to load  InfluxDB client configuration from environment variable")
```

Comment:
Fix double space typo in `load  InfluxDB`.

Reply:
fixed